db/migrate/20200428143000_create_network_service.rb

```diff
@@ -0,0 +1,29 @@
+# rake db:migrate:down VERSION=20200428000001
+# rake db:migrate:up VERSION=20200428000001
+class CreateNetworkService < ActiveRecord::Migration[5.0]
+  def change
+    create_table :network_services do |t|
+      t.string     :type
+      t.string     :name
+      t.string     :description
+      t.string     :ems_ref
+      t.references :ems,               :type => :bigint
+      t.references :cloud_tenant,      :type => :bigint   
```

Comment:
Could you add the created_at/updated_at columns as well to all these new tables?  `t.timestamps` will do it

Reply:
Added t.timestamps to network_services, network_service_entries, security_policies and security_policy_rules

Reply:
:+1: thanks